c7n/resources/ecs.py

```diff
@@ -294,6 +294,86 @@ class ServiceTaskDefinitionFilter(RelatedTaskDefinitionFilter):
              - type: stop
     """
 
+@ECSCluster.filter_registry.register('storage')
+class Storage(ValueFilter):
+    """Filter clusters by configured storage parameters.
+
+    :Example:
+
+    Find any ECS clusters that have instances that are using unencrypted EBS volumes.
+
+    .. code-block:: yaml
+
+        policies:
+          - name: encrypted-ebs-volumes
+            resource: ecs
+            filters:
+              - type: storage
+                key: Encrypted
+                value: true
+    """
+
+    schema = type_schema(
+        'storage', rinherit=ValueFilter.schema,
```

Comment:
We'll need to make sure this filter knows it has an `operator` param. Something like:

```suggestion
        'storage', rinherit=ValueFilter.schema,
        operator={'type': 'string', 'enum': ['or', 'and']},
```